static/css/main.css

```diff
@@ -1,6 +1,5 @@
 body {
     font-size: 0.9rem;
-    font-family: Arial, serif;
```

Comment:
I largely removed the changes to global files in the most recent commit. I had changed this specifically because the default bootstrap fonts have better control over font-weight property, and my static files used that for purely aesthetic reasons. It doesn't fix any problems or enhance the UI, so I removed it in the most recent commits.